src/test-utils/dom/steps/index.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { ComponentWrapper, ElementWrapper } from '@cloudscape-design/test-utils-core/dom';
+
+import statusIconStyles from '../../../status-indicator/styles.selectors.js';
+import testUtilStyles from '../../../steps/test-classes/styles.selectors.js';
+
+export class StepWrapper extends ComponentWrapper {
```

Comment:
This is WIP but any feedback regarding what selectors we should have is appreciated